Makefile.in

```
@@ -56,6 +58,11 @@ build-cgi: $(LIBRARY).lib xmlcgi.pgm
 
 build-test: $(TEST_PGMS) $(TEST_SRVPGMS)
 
+prod-clean:
+	rm -f *.lib *.pgm *.srvpgm *.module *.sqlinst *.srcpf
```

```suggestion
	rm -f *.module *.srcpf
```

Since you're only removing the file and module objects

---

@kadler I don't why last two lines were lost in the compare. I  I want to only leave 5 pgm/svrpgm in library QXMLSERV. This is my changes:
```
prod: build-main build-procedure build-cgi prod-clean

prod-clean:
        rm -f *.lib *.pgm *.srvpgm *.module *.sqlinst *.srcpf
        rm -rf /qsys.lib/$(LIBRARY).lib/*.FILE
        rm -rf /qsys.lib/$(LIBRARY).lib/*.MODULE

```
Could you add that in the Makefile.in if it looks good to you? Thanks!

---

They're there in the compare, I _suggested_ the above changes in my comment. There's no need to remove the `*.lib *.pgm *.srvpgm *.sqlinst` files if you're not removing those objects from the library.

---

FYI: https://github.blog/changelog/2018-10-16-suggested-changes/